desktop-app/app/actions/browser.js

```diff
@@ -329,6 +329,20 @@ export function goToHomepage() {
   };
 }
 
+export function gotoUrl(url) {
```

Comment:
Is it possible we can just re-use the exiting `onAddressChange` action?

Reply:
Yes totally I fixed it with the last commit